.github/workflows/e2e.yml

```diff
@@ -38,16 +38,15 @@ jobs:
       working-directory: .github
       run: |
         git init
-        git pull https://${{ github.repository_owner }}:${{ secrets.CR_PAT }}@github.com/scalar-labs/scalardl-samples.git 3.4
+        git pull https://${{ github.repository_owner }}:${{ secrets.CR_PAT }}@github.com/scalar-labs/scalardl-samples.git 3.5
         docker-compose -f docker-compose.yml -f expose-cassandra-port.yml up -d cassandra
         echo -n "Wait for C* "
         until docker exec scalardl-samples-cassandra-1 cqlsh 2> /dev/null ; do
           sleep 0.5
           echo -n "#"
         done
         echo ""
-        docker cp function_schema.cql scalardl-samples-cassandra-1:function_schema.cql
-        docker exec scalardl-samples-cassandra-1 cqlsh -f function_schema.cql
+        docker run --rm --network scalar-network -v "${PWD}/scalardb-schema-loader.properties":/scalardb.properties -v "${PWD}/schema.json":/schema.json ghcr.io/scalar-labs/scalardb-schema-loader:3.7.0 -c /scalardb.properties -f /schema.json
```

Comment:
Use scalardb-schema-loader to replace using C* CQL for creating the testing schema.

Reply:
Should we use 3.6 since scalardl 3.5 uses scalardb 3.6?

Reply:
@feeblefakie 
Sorry, my bad.
I should have used 3.6 for consistency.
I fixed it in https://github.com/scalar-labs/scalardl-node-client-sdk/pull/81/commits/3d55c5e4abb11dbd8881d300efebe9b913663067

Please take a look.
Thank you!